src/plotman/configuration.py

```diff
@@ -94,9 +94,18 @@ class Plotting:
 class UserInterface:
     use_stty_size: bool = True
 
+@attr.frozen
+class Interactive:
+    autostart_plotting: bool
+
+@attr.frozen
+class Tools:
+    interactive: Optional[Interactive] = None
```

Comment:
Could we just give a default value here?  Or do we need to explicitly know when this isn't specified.

```suggestion
@attr.frozen
class Interactive:
    autostart_plotting: bool = True

@attr.frozen
class Tools:
    interactive: Interactive = attr.ib(factory=Interactive)
```